charts/prometheus-elasticsearch-exporter/templates/deployment.yaml

```diff
@@ -48,10 +48,9 @@ spec:
 {{- end }}
       {{- include "elasticsearch-exporter.image.pullSecret.name" (dict "images" (list .Values.image) "context" $) | nindent 6 }}
       restartPolicy: {{ .Values.restartPolicy }}
-      {{- if .Values.securityContext.enabled }}
+      {{- with .Values.podSecurityContext }}
       securityContext:
-        runAsNonRoot: true
-        runAsUser: {{ .Values.securityContext.runAsUser }}
+        {{- toYaml . | nindent 8 }}
```

Comment:
@monotek: The pod level SecurityContext is set in the currently released Helm chart, I’m merely moving it from hardcoded in the deployment to "podSecurityContext" in values.yaml.